hanadb_exporter/exporters/prometheus_exporter.py

```diff
@@ -45,7 +45,7 @@ def _manage_gauge(self, metric, formatted_query_result):
                 # TODO: exception labels not found
                 # TODO: exception value not found
                 try:
-                    labels.insert(metric.labels.index(column_name), column_value)
+                    labels.insert(metric.labels.index(column_name), column_value.lower())
```

Comment:
Shouldn't we force metric.labels to be lower case as well? (in case that the values in the metrics.json is upper case, e.g. "Host/HOST" instead of 'host"), same thing for values...